x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.test.ts

```diff
@@ -100,6 +100,17 @@ describe('EnterpriseSearchRequestHandler', () => {
         });
       });
 
+      it('passes a body if that body is a string buffer', async () => {
```

Comment:
Added this for code coverage which was missing

Reply:
Nice catch!